app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.errors import CommandLineError
+
+
+def copy_file(command: str) -> None:
+
+    if len(command.split(" ")) != 3 or command.split(" ")[0] != "cp":
+        raise CommandLineError("Command line is incorrect.")
+
+    command_line, original_file, new_file = command.split(" ")
```

Comment:
Do not perform split multiple times, define a variable, write split to it, and after that you can assess length and unpack this variable